torchtune/modules/peft/lora.py

```diff
@@ -59,6 +59,7 @@ def __init__(
         self.use_bias = use_bias
         self._quantize_base = quantize_base
         weight, bias = self._create_weight_and_bias()
+        self.disabled = False # if to turn off LoRA
```

Comment:
Do you mind expanding on this comment to explain that this can be used in DPO for treating the lora adapters as the policy model and disabling it to treat the base model as the reference model?